### PR TITLE
Added hasOwn and fromEntries methods to Object

### PR DESCRIPTION
**Pull request by mansoor-s**
All Test262 tests are passing.

Re: undefined "next" method in the iteratorRecord, I wasn't sure if the error messages are standardized because I couldn't find anything about it in the specifications.

### CODE REVIEW

File: builtin_object.go [hunk 1]
```diff
@@ -556,6 +556,46 @@ func (r *Runtime) object_setPrototypeOf(call FunctionCall) Value {
 	return o
 }
 
+func (r *Runtime) object_fromEntries(call FunctionCall) Value {
+	o := call.Argument(0)
+	r.checkObjectCoercible(o)
+
+	result := r.newBaseObject(r.global.ObjectPrototype, classObject).val
+
+	adderFn := func(call FunctionCall) Value {
```
Comment (dop251) [inline builtin_object.go:565]: Why not inline this? Would also save the `.ToObject()` call.

**Comment by dop251** [comment]
Thanks for submitting! I've added a comment... As for the error message, indeed they are not specified, but what I tend to do it run a test case in node and copy the error message from there.

File: builtin_object.go (hunk 1 above)
Comment (mansoor-s) [reply builtin_object.go:565]: Good idea, done.

**Comment by dop251** [comment]
Merged in 936bcca

**Pull request closed without merging**